CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+### citus v11.0.5 (August 1, 2022) ###
+
+* Avoids possible information leakage about existing users
+
+* Allows using `WITH HOLD` cursors with parameters
+
+* Fixes a bug that could cause failures in `INSERT INTO .. SELECT`
+
+* Fixes a bug that prevents pushing down `IN` expressions when using columnar
+  custom scan
```

Comment:
made up this for https://github.com/citusdata/citus/pull/6030 as it didn't have a DESCRIPTION line